src/components/Loader.vue

```diff
@@ -130,25 +146,35 @@ class Loader extends Vue {
 
     // In a browser, setTimout returns a number, but Typescript thinks it should be a NodeJS.Timeout
     private stateUpdateTimeout: any = null;
+    private indicatorDisplayTimeout: any = null;
```

Comment:
when using `window.setTimeout`, it knows that you are using the browser verion. Can thus be typed as number.

Reply:
Cool!